samples/maps/demo/basic-map/demo.js

```diff
@@ -0,0 +1,73 @@
+const parseData = () => {
+    const lifeData = document.getElementById('csv').innerHTML;
+    const lines = lifeData.trim().split('\n');
+    lines.shift(); // Remove first line
+
+    const data = lines.map(line => {
+        const values = line.split(';');
+        if (values.length < 3 || isNaN(parseFloat(values[2]))) {
+            return null;
+        }
+        return {
+            code: values[1],
+            value: parseFloat(values[2])
+        };
+    }).filter(p => p); // Filter out null values
+    return data;
+};
+
+(async () => {
+
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/world.topo.json'
+    ).then(response => response.json());
+
+    Highcharts.mapChart('container', {
+        chart: {
+            map: topology
+        },
+
+        title: {
+            text: 'Life expectancy in the world (2021)'
+        },
+
+
+        credits: {
+            href: 'https://data.worldbank.org',
+            mapText: ' Data source: The World Bank'
+
+        },
+        mapNavigation: {
+            enabled: true,
+            buttonOptions: {
+                verticalAlign: 'bottom'
+            }
+        },
+
+        colorAxis: {
+            min: 60
+        },
+
+        data: {
+            seriesMapping: [{
+                code: 1,
+                value: 2
+            }]
+        },
```

Comment:
This block is not used. It belongs to the data module. Speaking of which, we could also parse this CSV with the data module, but it looks like you already tried that?

Reply:
I tried it, but must have messed something up because I couldn't get it to work! Works as a charm now 🎉